exercises/supervised-regression/ex_rnw/ex_lm_abalone.Rnw

```diff
@@ -0,0 +1,43 @@
+We want to predict the age of an abalone using its longest shell measurement and 
+its weight.
+
+See \url{http://archive.ics.uci.edu/ml/datasets/Abalone} for more details.
```

Comment:
Die haben eine neue Seite in beta (aber dafür https), vlt. nehmen wir besser die? https://archive-beta.ics.uci.edu/ml/datasets/abalone